doc/source/rucio_demo.rst

```diff
@@ -25,12 +118,25 @@ The bootstrap script creates a Rucio instance with 2 accounts (root and jdoe) an
   updated_at : 2018-02-08T15:37:26
   deleted_at : None
   email      : None
-  [root@ad03d8dc3b4a rucio]# rucio list-scopes
-  test
-  user.jdoe
 
-Testing dataset upload and creation of rules
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To enable Bash tab completion for the Rucio CLIs::
+
+ $ source /activate_rucio_global_completion.sh
```

Comment:
This is not necessary anymore. In #791 I added this to .bashrc. So it will load automatically.

Reply:
I changed it